WanderLost/WanderLost/Client/Components/Card.razor

```diff
@@ -9,16 +9,19 @@
     </div>
   </div>
   <div class="m-card__buttons">
-    <CardButton Type="CardButton.Variant.Like" OnClick="@(async () => await SubmitVote(Merchant.Id, VoteType.Upvote))" Active="@(CurrentVote(Merchant.Id) == VoteType.Upvote)" />
+    <CardButton ClassName="m-card__like" Type="CardButton.Variant.Like" OnClick="@(async () => await SubmitVote(Merchant.Id, VoteType.Upvote))" Active="@(CurrentVote(Merchant.Id) == VoteType.Upvote)" />
     <div class="m-card__score">@Merchant.Votes</div>
-    <CardButton Type="CardButton.Variant.Dislike" OnClick="@(async () => await SubmitVote(Merchant.Id, VoteType.Downvote))" Active="@(CurrentVote(Merchant.Id) == VoteType.Downvote)" />
+    <CardButton ClassName="m-card__dislike" Type="CardButton.Variant.Dislike" OnClick="@(async () => await SubmitVote(Merchant.Id, VoteType.Downvote))" Active="@(CurrentVote(Merchant.Id) == VoteType.Downvote)" />
+    <CardButton ClassName="m-card__report" Title="Replace" Type="CardButton.Variant.Report" OnClick="@OnClickReport" />
```

Comment:
Based on user vote existence title should be either "Submit correction" or "Suggest new location"

Reply:
Yeah, good idea, done